### PR TITLE
Changed Core Adapters primary URL

### PR DESCRIPTION
**Pull request by ZakAyesh**
Put old URL in firebase.json so backlinks still work.

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -22,6 +22,11 @@
         "source": "/docs/kovan-keeper-network-beta/",
         "destination": "/docs/chainlink-keepers/introduction/",
         "type": 301
+      },
+      {
+        "source": "/docs/adapters",
```
Comment (StephenFluin) [inline firebase.json:27]: Can you make sure these end with `/`?

**Review by StephenFluin — CHANGES_REQUESTED** [review]

**Comment by StephenFluin** [comment]
It looks like there are 3 commits right now. Can you squash them and force push the PR so there's only one that's based off of master?

File: firebase.json (hunk 1 above)
Comment (ZakAyesh) [reply firebase.json]: Completed and squashed changes into one commit.

**Comment by ZakAyesh** [comment]
> It looks like there are 3 commits right now. Can you squash them and force push the PR so there's only one that's based off of master?

Should be all good now

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**